treeproxyfilter.cpp

```diff
@@ -28,7 +28,7 @@ TreeProxyFilter::TreeProxyFilter(QObject *parent) : QSortFilterProxyModel(parent
 
 bool TreeProxyFilter::filterAcceptsRow(int sourceRow, const QModelIndex &sourceParent) const
 {
-	if (!filterRegExp().isEmpty())
+    if (filterRegularExpression().isValid())

```

Comment:
The change from tab to space indents here isn't _technically_ a problem, but it does make the alignment of this statement a bit "weird" relative to the surrounding code.

Reply:
Ah, I changed my editor to use tabs instead of spaces. Thanks for pointing that out.